api/models/round.py

```diff
@@ -0,0 +1,104 @@
+"""Models for Round"""
+from datetime import datetime
+from typing import List
+
+from pydantic import BaseModel
+from tests.sample_data.rounds import ROUNDS
+
+from .eligibility_criteria import EligibilityCriteria
+
+
+class Round(BaseModel):
+    """Pydantic model for rounds"""
+
+    fund_id: str
+    round_title: str
+    round_id: str
+    eligibility_criteria: EligibilityCriteria = None
+    opens: datetime
+    deadline: datetime
+    application_url: str
+
+    class Config:
+        schema_extra = {
+            "example": {
+                "fund_id": "funding-service-design",
+                "round_title": "Spring",
+                "round_id": "spring",
+                "eligibility_criteria": {"max_project_cost": 1000000},
+                "opens": "2022-02-01T00:00:01",
+                "deadline": "2022-06-01T00:00:00",
+                "application_url": (
+                    "https://application-form-service/fund-id-round-id"
+                ),
+            }
+        }
+
+    @classmethod
+    def cached_rounds(cls):
+        return ROUNDS
```

Comment:
So something im trying to mull over in my head but ill comment what im thinking - It is returning ROUNDS here from the import up top. Is this good practice or should it belong to the class as a class variable?

Reply:
Does this function just return the global variable ROUNDS? I assume this is done for future refactoring reasons.

Reply:
Will clarify in docstring